Recipe site - Patrick Kok

patrickkok opened the pull request.

patrickkok committed:
Initial commit, includes webpage link structure and recipe names

patrickkok committed:
added tomato egg recipe

Flashrob commented on chinese/chinese-cuisine.html:
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```
nice that you have 2 commits. If we make each page a separate commit, it would be even better

Flashrob commented on chinese/chinese-tomato-egg.html:
```diff
@@ -0,0 +1,137 @@
+  <body>
+    <img
+      src="https://christieathome.com/wp-content/uploads/2020/10/Chinese-Tomato-Egg-Stir-Fry-11-300x300.jpg"
+      alt=""
```
if the alt property is empty, might just not add it. Here it is a bit redundant

Flashrob commented on chinese/chinese-tomato-egg.html:
```diff
@@ -0,0 +1,137 @@
+      Chinese tomato egg stir-fry. Soft and silky eggs stir fried in a sweet
+      tangy tomato sauce with green onions. Ready in 15 minutes in one pan!
+    </p>
+    <table>
```
For now tables are fine, for the future let's not use them unless you really want a table specifically. Usually we use CSS to get a table-like layout on our pages. Tables are a very oldschool element :) Not a problem here, but just a hint and sth to consider in the future

Flashrob commented on chinese/chinese-tomato-egg.html:
```diff
@@ -0,0 +1,137 @@
+        <td align="right">1</td>
+        <td>green onion <font color="grey">finely chopped</font></td>
+      </tr>
+      <tr>
+        <td><br /><br /></td>
+      </tr>
+      <tr>
+        <td><h3>Instructions</h3></td>
+        <td><hr /></td>
+      </tr>
```
These two rows with 2 line breaks and one horizontal ruler seem a bit odd. Are those necessary?

Commit:
Add ingredients and instructions for `chinese-sesame-chicken.html`

Commit:
Update `chinese-tomato-egg.html`
- Add image width
- Reduce header weight from 3 to 4
- Add line breaks in instructions

Commit:
Add images to hyperlinks

Commit:
Add ingredients and instructions to `western-steak.html`

Commit:
Update html title for `western-steak.html and `western-cuisine.html`

Commit:
Add ingredients and instructions for `western-bolognese.html`

patrickkok committed:
Remove breakline and empty table row, remove alt attribute

patrickkok committed:
Remove alt text in all image tags

patrickkok committed:
Add preperation time to all recipes
